package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "glue",
   "description": "Server composer for hapi.js",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
You'll probably have to revert that change anyway. I can't promise it'll be faster as I don't know if @csrl is around.